packages/eth-json-rpc-provider/src/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from './provider-from-engine';
 export * from './provider-from-middleware';
-export type { SafeEventEmitterProvider } from './safe-event-emitter-provider';
```

Comment:
This changes what is being exported (the whole class instead of just the type). What's the reason for this change?

Reply:
See: first line of `network-controller/tests/fake-provider`.

Reply:
Okay, no problem. I guess this wouldn't be a breaking change, so it's okay.